packages/wallet/backend/src/rafiki/backend/generated/graphql.ts

```diff
@@ -763,6 +803,11 @@ export type PaymentPointersConnection = {
   pageInfo: PageInfo;
 };
 
+export enum PaymentType {
+  Incoming = 'INCOMING',
```

Comment:
Well, I think UPPERCASE is better? like INCOMING = 'INCOMING'

Reply:
this file is generated from rafiki graphql schema